components/ItemSummary.tsx

```diff
@@ -32,8 +34,10 @@ export default function ItemSummary(props: ItemSummaryProps) {
         </span>
       </div>
       <div class="text-gray-500">
-        {pluralize(props.item.score, "point")} by{" "}
-        {getUserDisplayName(props.user)}{" "}
+        <span id={`score-${props.item.id}`}>
+          {pluralize(props.item.score, "point")}
+        </span>{" "}
```

Comment:
This `<span />` with `id` attribute can be removed. See below.

Reply:
I would remove this with `changePoint()` in `VoteButton.tsx` 

The point would not update though after clicking the `VoteButton` after removing this.

ps: You can access this code in the file history if needed.